OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/EditorTopComponent.java

```diff
@@ -253,11 +248,11 @@ public void setCurrentFile(File currentFile) {
     }
 
     public DbxState getDbxState() {
-        return dbxState;
+        return getEditorState().getDbxState();
     }
 
     public void setDbxState(DbxState dbxState) {
-        this.dbxState = dbxState;
+        getEditorState().setDbxState(dbxState);

```

Comment:
We should remove all of those getters/setters from this class and only have one for EditorState instance.